change/@fluentui-react-popover-e8ab1f66-cf35-4308-a6ee-45f3788a6195.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32022&commitId=9efb5fd921a439344f41c3dfe7893e12ada8e3be&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Subtle.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Subtle.selected.chromium.png|0| Removed|

</details>


  <details><summary><b>Popover Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Popover Converged.positioning - RTL.positioned popovers.chromium.png|7| Changed|
|Popover Converged.positioning - High Contrast.positioned popovers.chromium.png|4| Changed|
|Popover Converged.positioning.positioned popovers.chromium.png|7| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>